bdsx/bds/player.ts

```diff
@@ -883,7 +923,23 @@ export class ServerPlayer extends Player {
 }
 
 export class SimulatedPlayer extends ServerPlayer{
+    /**
+     * Create SimulatedPlayer
+     * @param name
+     * @param blockPos
+     * @param dimensionId
+     * @param nonOwnerPointerServerNetworkHandler Minecraft.getNonOwnerPointerServerNetworkHandler()
+     */
+    create(name: string, blockPos: BlockPos, dimensionId: DimensionId, nonOwnerPointerServerNetworkHandler: Bedrock.NonOwnerPointer<ServerNetworkHandler>): SimulatedPlayer{
```

Comment:
I am not sure whether `Bedrock.NonOwnerPointer<ServerNetworkHandler>` is actually a singleton or not, if it is, i think the `nonOwnerPointerServerNetworkHandler` param should be removed and provide the param internally to the native function.
Just my thoughts, see how others think about it

Reply:
`bedrockServer.serverNetworkHandler` is not accepted. I see the symbol requires `class gsl::not_null<class Bedrock::NonOwnerPointer<class ServerNetworkHandler>>`.